mjpc/tasks/humanoid/humanoid.cc

```diff
@@ -33,29 +33,18 @@ namespace mjpc {
 // ----------------------------------------------------------------
 void Humanoid::ResidualStand(const double* parameters, const mjModel* model,
                              const mjData* data, double* residual) {
-  int counter = 0;
+    // ----- action ----- //
```

Comment:
As discussed offline, these changes should be minimal, rather than a full rollback, so we understand what it is that makes this cost better behaved.

Reply:
Task now in sync with main. Minor change to balance norm 1 -> 6 to get good behaviour.